exonum-light-client/src/main/lombok/com/exonum/client/response/Block.java

```diff
@@ -80,8 +80,11 @@
   ZonedDateTime commitTime;
 
   /**
-   * Returns time when the block was committed to the blockchain.
-   * Can be empty if include time parameter is not specified in the request.
+   * Returns the time when the block was committed to the blockchain.
```

Comment:
`when the block is committed`?

Reply:
Why present, aren't they all committed in the past?

Reply:
You are right, I misunderstood the sentence.